lib/docblockr-worker.js

```diff
@@ -32,9 +33,9 @@ module.exports =
         self.event = event;
         var regex = {
           // Parse Command
-          'parse': /^\s*(\/\*|###)[*!]\s*$/,
+          'parse': /^\s*((\/\*|###)[*!]|\/\/\*)\s*$/,
```

Comment:
I couldn't find anything about this `///*` pattern to start doc comments in sassdoc. Am I missing something? If not, could we make it work with just `///`?

Reply:
it was `//*` as a personal preference, but I agree `///` would be better